src/project_setup.jl

```diff
@@ -254,17 +200,31 @@ The new project remains activated for you to immidiately add packages.
 * `force = false` : If the `path` is _not_ empty then throw an error. If however `force`
   is `true` then recursively delete everything in the path and create the project.
 * `git = true` : Make the project a Git repository.
-* `placeholder = false` : Add hidden place holder files in each default folder to ensure that project
-  is maintained when the directory is cloned. Should be used only when `git = true`.
+* `template = DrWatson.DEFAULT_TEMPLATE` : A template containing the folder structure
+  of the project. It should be a vector containing strings (folder) or pairs of `String
+  => Vector{String}`, containg a folder and subfolders (this can be nested further). Example:
+  ```
+  DEFAULT_TEMPLATE = [
+    "_research", 
+    "src", 
+    "scripts",
+    "plots", 
+    "notebooks",
+    "papers",
+    "data" => ["sims", "exp_raw", "exp_pro"],
+  ]
+  ```
+* `placeholder = false` : Add hidden place holder files in each default folder to ensure 
+  that project folder structure is maintained when the directory is cloned.
+  Should be used only when `git = true`.
   Will throw a warning if used with `git = false`
 """
 function initialize_project(path, name = default_name_from_path(path);
-    force = false, readme = true, authors = nothing,
-    git = true, placeholder = false)
-
-    placeholder && !git && @warn "The placeholder files are created, but you need to "*
-    "manually commit them to your version control software OR initialize project with git = true"
+        force = false, readme = true, authors = nothing,
+        git = true, placeholder = false, template = DEFAULT_TEMPLATE
+    )
 
+    if git == false; placeholder = false; end
```

Comment:
What's this doing? Did you deliberately remove the warning for placeholder == true, git == false

Reply:
Yes. Well, as we say in the documentation string, `placeholder` has no reason to be `true` if `git` is `false`. So I just thought of making it `false` always if this is the case.

If this is the wrong way to go let me know and I'll revert.

Reply:
I'm good with that. The docstring still contains the info about the warning though.